packages/plugin-board/test/integration/spec/board.js

```diff
@@ -0,0 +1,414 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import '../..';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import testUsers from '@ciscospark/test-helper-test-users';
+import {flaky} from '@ciscospark/test-helper-mocha';
+import fh from '@ciscospark/test-helper-file';
+import {find, map} from 'lodash';
+import uuid from 'uuid';
+
+function generateTonsOfContents(numOfContents) {
+  const contents = [];
+
+  for (let i = 0; i < numOfContents; i++) {
+    contents.push({
+      type: `curve`,
+      payload: JSON.stringify({id: i, type: `curve`})
+    });
+  }
+  return contents;
+}
+
+function boardChannelToMercuryBinding(channelId) {
+  // make channelId mercury compatible replace `-` with `.` and `_` with `#`
+  return channelId.replace(/-/g, `.`).replace(/_/g, `#`);
+}
+
+describe(`plugin-board`, function() {
+  this.timeout(30000);
+
+  let board, conversation, fixture, participants;
+  const mercuryBindingsPrefix = `board.`;
+
+  // create users
+  before(() => testUsers.create({count: 3})
+    .then((users) => {
+      participants = users;
+
+      return Promise.all(map(participants, (participant) => {
+        participant.spark = new CiscoSpark({
+          credentials: {
+            authorization: participant.token
+          }
+        });
+        return participant.spark.device.register();
+      }));
+    })
+
+    // create conversation
+    .then(() => participants[0].spark.conversation.create({
+      displayName: `Test Board Conversation`,
+      participants
+    }))
+    .then((c) => {
+      conversation = c;
+      return conversation;
+    })
+
+    // create channel (board)
+    .then(() => participants[0].spark.board.createChannel({aclUrl: conversation.id}))
+    .then((channel) => {
+      board = channel;
+      return channel;
+    })
+
+    // connect to realtime channel
+    .then(() => {
+      const mercuryBindingId = boardChannelToMercuryBinding(board.channelId);
+      const bindingStr = [mercuryBindingsPrefix + mercuryBindingId];
+      const bindingObj = {bindings: bindingStr};
+
+      return Promise.all(map(participants, (participant) => {
+        return participant.spark.board.register(bindingObj)
+          .then((url) => {
+            participant.spark.board.realtime.set({
+              boardWebSocketUrl: url.webSocketUrl,
+              boardBindings: bindingStr
+            });
+            return participant.spark.board.realtime.connect();
+          });
+      }));
+    })
+
+    // load fixture image
+    .then(() => fh.fetch(`sample-image-small-one.png`))
+
+    .then((fetchedFixture) => {
+      fixture = fetchedFixture;
+      return fetchedFixture;
+    })
+
+    // confirm we are listening
+    .then(() => Promise.all(map(participants, (participant) => {
+      assert(participant.spark.board.realtime.connected, `participant is not listening`);
```

Comment:
through out the rest of the sdk, the helper text correponds to the desired state, not the failure state. also, assert.isTrue might be a slightly easier to read error message.